treeherder/webapp/api/resultset.py

```diff
@@ -133,6 +133,22 @@ def cancel_all(self, request, project, jm, pk=None):
         except Exception as ex:
             return Response("Exception: {0}".format(ex), 404)
 
+    @action(permission_classes=[IsAuthenticated])
+    @with_jobs
+    def fill_all(self, request, project, jm, pk=None):
```

Comment:
I think it would be better to use a more descriptive name for this action. Why don't you use 'trigger_missing_jobs' instead? Also, could you please add some more context to the doctstring?